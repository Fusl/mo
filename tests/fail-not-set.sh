#!/usr/bin/env bash

cd "${0%/*}"
unset __NO_SUCH_VAR
echo "This will fail: {{__NO_SUCH_VAR}}" | ../mo --fail-not-set 2>&1

if [[ $? -ne 1 ]]; then
    echo "Did not return 1"
fi
