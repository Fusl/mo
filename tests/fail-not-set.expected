Populated: words;
Empty: ;
Unset: Env variable not set: __NO_SUCH_VAR
